frontend/src/components/HostMetricsDetail/Containers.tsx

```diff
@@ -0,0 +1,55 @@
+import './Containers.styles.scss';
+
+import { Button, Space, Typography } from 'antd';
+import { LifeBuoy } from 'lucide-react';
+import { handleContactSupport } from 'pages/Integrations/utils';
+import { isCloudUser } from 'utils/app';
+
+const { Text } = Typography;
+
+function Containers(): JSX.Element {
+	return (
+		<Space
+			direction="vertical"
+			style={{ maxWidth: '600px', margin: '64px auto', padding: '0 16px' }}
+			size={24}
+		>
+			<div className="infra-container-card">
+				<img
+					src="/Icons/infraContainers.svg"
+					alt="infra-container"
+					width={32}
+					height={32}
+				/>
+				<Text className="infra-container-card-text">
+					The ability to visualise containers is in active development and should be
+					available to you soon.
+				</Text>
+			</div>
+
+			<div className="infra-container-working-msg">
+				<Space>
+					<img src="/Icons/broom.svg" alt="broom" width={16} height={16} />
+					<Text className="infra-container-card-text">
+						We&apos;re working to extend infrastructure monitoring to take care of a
```

Comment:
do we need `&apos;`? 

```suggestion
						We're working to extend infrastructure monitoring to take care of a
```